plugins/CoreHome/Tracker/VisitRequestProcessor.php

```diff
@@ -155,8 +155,9 @@ public function afterRequestProcessed(VisitProperties $visitProperties, Request
          * This event is primarily used by the **PrivacyManager** plugin to anonymize IP addresses.
          *
          * @param string &$ip The visitor's IP address.
+         * @param mixed $idSite The site ID we're tracking the visit for.
```

Comment:
```suggestion
         * @param int $idSite The site ID we're tracking the visit for.
```

At this point we know we have a site,. And with the trace through the various request functions "getIdSiteIfExists -> getIdSite -> getIdSiteUnverified -> getRequestVar(int)" this should always be an int. Though I would keep the int cast until PHPStan tells us the cast is definitely unnecessary 🙈

Reply:
Yep, changed to int